docs/handbook/getting-started.md

```diff
@@ -281,17 +307,18 @@ If you wish to purchase a ready made DVD or USB drive there is also [OSDISC.COM]
 Download example:
 
 ```
-wget "http://dlc.openindiana.org/isos/hipster/OI-hipster-gui-20161030.iso"
-wget "http://dlc.openindiana.org/isos/hipster/OI-hipster-gui-20161030.iso.sha256sum"
+$ wget "http://dlc.openindiana.org/isos/hipster/OI-hipster-gui-20161030.iso"
+$ wget "http://dlc.openindiana.org/isos/hipster/OI-hipster-gui-20161030.iso.sha256sum"
 ```
 
 ### Checking the MD5/SHA
 
 Checksum verification example:
 
 ```
-sha256sum --check OI-hipster-gui-20161030.iso.sha256sum
+$ sha256sum --check OI-hipster-gui-20161030.iso.sha256sum
 OI-hipster-gui-20161030.iso: OK
+$
```

Comment:
Yes, to show that the prompt has returned. Did it in a few other places as well.